docs/CONTRIBUTING.md

```diff
@@ -1,31 +0,0 @@
-# Documentation Setup
```

Comment:
Integrate with DEVELOPING.md. 

Revert this delete. In the PR for developing.md, delete this and ask (in the pr description) that its content get integrated.